### ISSUE TITLE
 High Contrast: Keyboard focus is not clearly visible for the buttons in the "Calendar".

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [OfficeDev/office-ui-fabric-react#5499]
### Bug Report 
 
- __Package version(s)__: 5.112.1003
- __Browser and OS versions__: Edge with HC Black and White

Products/sites affected: VSTS
 
#### Describe the issue:
 
While navigating the buttons/days/months etc in the Calendar, keyboard focus should be clearly visible on them with a distinguishable border.
 
### If applicable, please provide a codepen repro:
 The issue is there on official office fabric calendar home page as well: https://developer.microsoft.com/en-us/fabric#/components/Calendar#Overview

Here is the snapshot
![image](https://user-images.githubusercontent.com/24951429/42492835-308a81d6-8438-11e8-8618-f26ff8425b1e.png)

**Comment by natalieethell** [comment]
Hey @AkshanshSrivastava03 thanks for submitting this issue! It looks like this is due to wrapping the day in an additional span which was added here: https://github.com/OfficeDev/office-ui-fabric-react/pull/1821/files#diff-ceb54624d97a7b7fd449c017efad5195R170. 

@mdahamiwal I noticed that in your PR above, you were fixing some accessibility issues. Would Calendar maintain its accessibility if the `span` wrapping the day is removed? Just experimenting with Narrator on my own, it seems to read off the same information, but it'd be great to have that confirmed.

**Comment by AkshanshSrivastava03** [comment]
@natalieethell - When can we expect this issue to be resolved? 
We are using Calendar in our product and are hitting this issue under High contrast Compliance.

**Comment by natalieethell** [comment]
Sorry for the delay here! @AkshanshSrivastava03 are you still experiencing this issue? Here's what I'm seeing in HC Black and HC White.

![calendardayhcblackworking](https://user-images.githubusercontent.com/14134000/46772160-71788d80-ccab-11e8-8a6f-a59e8fe08b59.gif)

![calendardayhcblackandwhiteworking](https://user-images.githubusercontent.com/14134000/46772067-1777c800-ccab-11e8-89d1-0da38f459bab.gif)

**Comment by natalieethell** [comment]
I'm going to close this issue, but @AkshanshSrivastava03 feel free to reopen it if you are still having problems.
